ivre/utils.py

```diff
@@ -1859,7 +1861,12 @@ def parse_ssh_key(data: bytes) -> Dict[str, Any]:
             # convert integer to strings to prevent overflow errors
             # (e.g., "MongoDB can only handle up to 8-byte ints")
             for val in ["exponent", "modulus"]:
-                info[val] = str(info[val])
+                try:
+                    # with large SSH keys, it could lead to an exception
+                    info[val] = str(info[val])
+                except ValueError:
+                    sys.set_int_max_str_digits(int(math.log(info[val], 10) + 1))
+                    info[val] = str(info[val])
```

Comment:
Since this is new in Python 3.11, and since we might want to preserve the original value (I'm not sure on this part, I have no idea of the consequences of calling `sys.set_int_max_str_digits()`), we could do:
```suggestion
                except ValueError as exc:
                    try:
                        orig = sys.set_int_max_str_digits()
                    except AttributeError:
                        raise exc
                    sys.set_int_max_str_digits(int(math.log(info[val], 10) + 1))
                    info[val] = str(info[val])
                    sys.set_int_max_str_digits(orig)                    
```
or:
```suggestion
                except ValueError as exc:
                    try:
                        sys.set_int_max_str_digits(int(math.log(info[val], 10) + 1))
                    except AttributeError:
                        raise exc
                    info[val] = str(info[val])
```
WDYT?